refinery/tool_manager/tests.py

```diff
@@ -3648,6 +3651,22 @@ def assertions(tool):
             assertions
         )
 
+    def test_input_node_limit(self):
+        with mock.patch("constants.REFINERY_SOLR_DOC_LIMIT", 10):
+            tool = self.create_tool(ToolDefinition.VISUALIZATION)
+            tool_launch_config = self.tool.get_tool_launch_config()
+
+            # Crete one more entry than what REFINERY_SOLR_DOC_LIMIT permits
+            tool_launch_config[Tool.FILE_RELATIONSHIPS] = str(
+                [uuid.uuid4()] * 11
```

Comment:
`(constants.REFINERY_SOLR_DOC_LIMIT + 1)` instead? And then the comment probably isn't needed.

Reply:
I ran into trouble running this test quickly (if at all) when making an array that large. `constants.REFINERY_SOLR_DOC_LIMIT + 1` -> 10000001

I think it was probably trying to write something that big to the db

How about something like:
```
with mock.patch("constants.REFINERY_SOLR_DOC_LIMIT", 10) as solr_limit:
    tool = self.create_tool(ToolDefinition.VISUALIZATION)
    tool_launch_config = self.tool.get_tool_launch_config()

    # Crete one more entry than what REFINERY_SOLR_DOC_LIMIT permits
    tool_launch_config[Tool.FILE_RELATIONSHIPS] = str(
        [uuid.uuid4()] * (solr_limit + 1)
    )
```

Reply:
If that's the case then `REFINERY_SOLR_DOC_LIMIT` should also be set to a lower number in `constants.py` to prevent potential outages.